src/Units/DoRWeapons.java

```diff
@@ -353,35 +363,35 @@ public CarrierMGun()
-/* DUSTERMGUN       */{  55,   45,    45,    18,    15,        5,    8,       5,        1,        15,        5,      20,        20,  15,      40,     45,       45,     55,       75,       90,       0,       0,   0,       0,       0,          0,      0 }, /* DUSTERMGUN       */
-/* COPTERROCKETS    */{   0,    0,     0,    75,    75,       10,   70,      45,       35,        65,       20,      75,        55,  70,       0,      0,        0,      0,        0,        0,      85,       5,  25,       0,      25,         25,     25 }, /* COPTERROCKETS    */
-/* COPTERMGUN       */{  75,   65,    65,    30,    30,        1,    8,       8,        1,        25,        1,      35,        25,  20,       0,      0,        0,      0,       65,       85,       0,       0,   0,       0,       0,          0,      0 }, /* COPTERMGUN       */
-/* GUNBOATGUN       */{   0,    0,     0,     0,     0,        0,    0,       0,        0,         0,        0,       0,         0,   0,       0,      0,        0,      0,        0,        0,      75,      40,  40,       0,      40,         40,     55 }, /* GUNBOATGUN       */
-/* CRUISERTORPEDOES */{   0,    0,     0,     0,     0,        0,    0,       0,        0,         0,        0,       0,         0,   0,       0,      0,        0,      0,        0,        0,      85,      28,  95,      95,      38,         38,     40 }, /* CRUISERTORPEDOES */
-/* CRUISERMGUN      */{   0,    0,     0,     0,     0,        0,    0,       0,        0,         0,        0,       0,         0,   0,     105,    105,      105,    105,      120,      120,       0,       0,   0,       0,       0,          0,      0 }, /* CRUISERMGUN      */
-/* SUBTORPEDOES     */{   0,    0,     0,     0,     0,        0,    0,       0,        0,         0,        0,       0,         0,  55,       0,      0,        0,      0,        0,        0,     120,      20,  55,      55,     110,         80,     85 }, /* SUBTORPEDOES     */
-/* CARRIERMGUN      */{   0,    0,     0,     0,     0,        0,    0,       0,        0,         0,        0,       0,         0,  55,      35,     35,       40,     40,       45,       55,       0,       0,   0,       0,       0,          0,      0 }, /* CARRIERMGUN      */
-/* BATTLESHIPCANNON */{  75,   70,    70,    70,    70,       65,   65,      50,       40,        70,       55,      75,        75,  65,       0,      0,        0,      0,        0,        0,      95,      65,  65,      50,       0,         45,     75 }  /* BATTLESHIPCANNON */
+// defenders:      INFANTRY, MECH,  BIKE, RECON, FLARE, ANTI_AIR, TANK, MD_TANK, WAR_TANK, ARTILLERY, ANTITANK, ROCKETS, MOBILESAM, RIG, FIGHTER, BOMBER, SEAPLANE, DUSTER, B_COPTER, T_COPTER, GUNBOAT, CRUISER, SUB, SUB_SUB, CARRIER, BATTLESHIP, LANDER, METEOR
```

Comment:
AWBW just uses NeoTank defense, and DoR has specific values? Interesting.

Reply:
Yeah, and IIRC AW2 actually uses Md Task defense.
Go figure.

Reply:
Then--and I don't care either way--why use neo defense here?

Reply:
`AWBWUnits.java`, really.
The immediate use case is AWBW players who want to playtest stuff, so that's the first thing I'm gonna support.

Reply:
What? Somehow your explanations often leave me more confused. `AWBWUnits.java` has stats for Md Tanks as well.

Reply:
Yes, but it's called `AWBWUnits.java`, not `AW2Units.java`.
If I'm gonna simulate something, I might as well do it right ~~unlike AWBW~~